### ISSUE TITLE
incorporating TES changes 

### ISSUE THREAD

**Issue by docrostov** [loathers/TourGuide#101]
TES has sent me a ton of changes he integrated in his personal tourguide that he never posted (or got missed) in the discord channel. this is going to be my generic issue i will keep the list in as i work on em

here are the changes i have successfully incorporated in the new PR:
> -include survival knife in desert exploration
-kitchen drawers should be int, not float (18666)
-add "drawers remaining to kitchen drawers (18692)
-CI omega button
-science tentacle should redirect to place.php?whichplace=forestvillage&action=fv_scientist instead of just place.php?whichplace=forestvillage
-small stinky cheese update for invsearch
-teleportitis/feeling lost warning + sgeea
-new banish sources and their durations
-monorail specific location
-the megafam drop tile https://github.com/uthuluc/TourGuide/blob/master/iotm2014buddybjorn
-protopack nag
-kgb red text while not equipped
-sausage grinder minor poke
-docbag invsearch
-# of lectures used for prof
-powerful glove battery charge should be named
-add exp% to commerce statgain estimate
-cmc missing "x advs until next consultation"
-cookbat cooking reminders

here are ones that are missing still:
>-drunk pygmy fights can get a resource reminder
-cubeling wants an x/12 progress tracker
-weird kramco issue where it's taking you to invsearch instead of grind.php
-LT&T supernag

here are the ones i am not adding (will explain in PR):
>-boombox is in resources instead of tasks, blocking its supernag at 1 fight remaining
-mcb is resource instead of task, blocking its supernag as well
-missing olfaction+turtlesex tile
-cookbookbat recipe reminder; they are now like 500 meat in the mall, so not worth adding

**Comment by uthuluc** [comment]
cookbookbat recipe + cooking reminder:
```    if (!get_property_boolean("_cookbookbatRecipeDrops"))
    {
        description.listAppend(HTMLGenerateSpanFont("Recipe of Before Yore available!", "blue"));
        resource_entries.listAppend(ChecklistEntryMake("__familiar cookbookbat", url, ChecklistSubentryMake("Pizza party with the Cookbookbat!", "", description)).ChecklistEntrySetIDTag("Cookbookbat Resource"));
    }
    int cookings_remaining = clampi(5 - get_property_int("_cookbookbatCrafting"), 0, 5);
    if (cookings_remaining > 0) 
    {
        description.listAppend(HTMLGenerateSpanOfClass(cookings_remaining, "r_bold") + " free cookings: Unstable fulminate, potions, and more.");
    }
```

insert this after the homebodyl free_crafts_left part of the crafting tile:
```        int free_cooks_left = 0;
        if (lookupFamiliar("Cookbookbat").familiar_is_usable()) {
            string [int] description;
            free_cooks_left += clampi(5 - get_property_int("_cookbookbatCrafting"), 0, 5);
            string title = "free cooking";
            craft_entry.subentries.listAppend(ChecklistSubentryMake(pluralise(free_cooks_left, title, title + "s") + " remaining", free_crafts_left > 0 ? "COOKING only" : "", description));
        }
```

**Comment by docrostov** [comment]
issue closed, largely resolved in #113 and what isn't resolved is in #115
